.gitattributes

```
@@ -1,2 +1,2 @@
 # We vendor typeshed from https://github.com/python/typeshed
-mypy/typeshed/ linguist-vendored
+mypy/typeshed/**/*.pyi linguist-vendored
```

```suggestion
mypy/typeshed/** linguist-vendored
```
There are a small handful of non pyi files in there